scmg.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2024 Valentin D'Emmanuele. All rights reserved.
+// Use of this source code is governed by a MIT-style license that can be
+// found in the LICENSE file.
+
+package sccp
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io"
+)
+
+// MsgType is type of SCMG message.
+type SCMGType uint8
+
+const (
+	_ SCMGType = iota
+	SCMGTypeSSA
+	SCMGTypeSSP
+	SCMGTypeSST
+	SCMGTypeSOR
+	SCMGTypeSOG
+	SCMGTypeSSC
+)
+
+// SCMG represents a SCCP Management message (SCMG).
+// ITU Q.713
```

Comment:
Unnecessary, as it's the same for any other SCCP. Or maybe you can add the section number which gives some more meaning.